### PR TITLE
Removed line translation if the original language is the same as the translation language

### PR DESCRIPTION
**Pull request by ZeDarkAdam**
Removed line translation if the original language is the same as the translation language to avoid displaying the same lines twice.
Screenshot 2024-06-09 212459

Fixed the dropdown text color being the same as the background.
Before:
Screenshot 2024-06-09 202731

After:
image

### CODE REVIEW

File: src/index.css [hunk 1]
```diff
@@ -36,7 +36,7 @@
 }
 
 .lyrics--translated {
-  opacity: 0.5;
+  opacity: 0.7;
```
Comment (boidushya) [inline src/index.css]: I think it looks better at 0.5

File: src/index.js [hunk 2]
```diff
@@ -234,18 +235,20 @@ const injectLyrics = (lyrics, wrapper) => {
       let target_language = items.translationLanguage || "en"; // Use the saved language or the default 'en'
 
       if (item.words.trim() !== "♪" && item.words.trim() !== "") {
-        translateText(item.words, target_language).then((translatedText) => {
-          if (translatedText) {
-            // If the translation was successful, set the translated text as the content for translatedLine
-            translatedLine.textContent = "\n" + translatedText;
+        translateText(item.words, target_language).then((result) => {
+          if (result) {
+            if (result.originalLanguage !== target_language) {
```
Comment (boidushya) [inline src/index.js]:
```suggestion
          if (result && result.originalLanguage !== target_language) {
```

File: src/index.js [hunk 3]
```diff
@@ -234,18 +235,20 @@ const injectLyrics = (lyrics, wrapper) => {
...
+              // If the translation was successful, set the translated text as the content for translatedLine
+              translatedLine.textContent = "\n" + result.translatedText;
+              line.appendChild(translatedLine);
+            }
```
Comment (boidushya) [inline src/index.js]:
```suggestion
           
```

**Review by boidushya — CHANGES_REQUESTED** [review]

File: src/index.css (hunk 1 above)
Comment (ZDAVanO) [reply src/index.css]: 0.5 is too difficult to see

File: src/index.js (hunk 2 above)
Comment (ZDAVanO) [reply src/index.js]: this causes a translation error to be thrown
image

Comment (boidushya) [reply src/index.js]: Ah makes sense! all good

File: src/index.css (hunk 1 above)
Comment (boidushya) [reply src/index.css]:
```suggestion
  opacity: 0.6;
```
Lets keep it at 0.6 then

**Review by boidushya — CHANGES_REQUESTED** [review]

**Review by boidushya — APPROVED** [review]

**Pull request merged**